src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/personalrelationships/profiledetails/ContactPersonProfileDetailsSyncService.kt

```diff
@@ -60,12 +61,14 @@ class ContactPersonProfileDetailsSyncService(
         ?: throw DomesticStatusChangedException("No ${profileType.name} profile type found for bookingId $bookingId")
       val (lastModifiedTime, lastModifiedBy) = profileDetails.lastModified()
 
-      getIgnoreReason(nomisResponse.bookings.size, booking.latestBooking, bookingId, profileDetails)
-        ?.let { ignoreReason ->
-          telemetry["reason"] = ignoreReason
-          telemetryClient.trackEvent("contact-person-${profileType.telemetryName}-synchronisation-ignored", telemetry)
-          return
-        }
+      if (!forceSync) {
+        getIgnoreReason(nomisResponse.bookings.size, booking.latestBooking, bookingId, profileDetails)
+          ?.let { ignoreReason ->
+            telemetry["reason"] = ignoreReason
+            telemetryClient.trackEvent("contact-person-${profileType.telemetryName}-synchronisation-ignored", telemetry)
+            return
+          }
+      }
```

Comment:
When recovering from a move booking we need to force the sync to DPS, even if the record was originally created or modified in DPS.